docker-compose-separate-containers.yml

```diff
@@ -1,23 +1,25 @@
-version: '2'
+version: '3'
```

Comment:
You're right, no reason to rush to v3 until other Stack compatibility issues are fixed.